pundix/assetlist.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "Pundi X Chain",
+  "assets": [
+    {
+      "description": "Pundi X Token",
+      "denom_units": [
+        {
+          "denom": "ibc/55367b7b6572631b78a93c66ef9fdfce87cde372cc4ed7848da78c1eb1dcdd78",
+          "exponent": 0
+        },
+        {
+          "denom": "PUNDIX",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/55367b7b6572631b78a93c66ef9fdfce87cde372cc4ed7848da78c1eb1dcdd78",
+      "name": "Pundi X Token",
+      "display": "PUNDIX",
+      "symbol": "PUNDIX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/pundi-x-token-logo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/pundi-x-token-logo.svg"
+      },
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "fxcore",
+            "base_denom": "eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
```

Comment:
This token isn't registered to [fxcore's assetlist](https://github.com/cosmos/chain-registry/blob/master/fxcore/assetlist.json). Please add there.